### DISCUSSION TITLE
What is the current next.js release checklist and process?

### DISCUSSION THREAD

**Discussion by shaunwarman** [vercel/next.js#55336] [Help] [answered]
### Summary

Is there a documented next.js release checklist and process? It would be great to get some idea of when the next stable semver will be cut. I see a large amount of effort and contribution go into canary releases and eventually a stable semver is released. How / why / when that final stable version is cut may be nextjs team decision, but wondering if you have a formula to help in the decisions there.

Similarly, is the roadmap and scope of focus publicly available?

**Comment by shaunwarman** [comment]
cc @leerob @timneutkens

**Comment by leerob** [comment, marked as answer]
We publish stable Next.js releases (promoting from `canary`) when we feel comfortable with the stability and reliability of the code that has landed so far. We like to err on releasing _more_ often, so there's usually many canary releases (sometimes multiple daily), lots of patches, fewer minor releases, and then 1-2 major releases per year typically.

This allows us to ship code frequently (and unblock folks by being able to update to `next@canary`) as well as trying to quickly land patches, fixes, and small improvements in patch releases. For minor releases, we always write a blog post to accompany the changes and summarize the work that's happened between each minor release. Those typically happen every 3 months, but could be faster or longer. It really depends on what features or improvements are in scope.

Hope this helps!

**Reply by shaunwarman** [reply]
Thanks @leerob ! The canaries help us. We're building up a regression suite against each release here. If we see issues, is the bug template / issue the best way to communicate problems? 

Looking forward to stable releases and especially the next one!

**Reply by leerob** [reply]
Yes, the issue process (with reproductions) is greatly appreciated!
